pkg/nsx/services/ipaddressallocation/ipaddressallocation.go

```diff
@@ -106,6 +106,7 @@ func (service *IPAddressAllocationService) Apply(nsxIPAddressAllocation *model.V
 	ns := service.GetIPAddressAllocationNamespace(nsxIPAddressAllocation)
 	VPCInfo := service.VPCService.ListVPCInfo(ns)
 	if len(VPCInfo) == 0 {
+		log.Error(nil, "Failed to list VPCInfo for IPAddressAllocation")
```

Comment:
how about move 109 after 110 to use the err in log.Error(err, ... ), using log.Error with error value nil seems weired.

Reply:
Updated, thanks!